src/entropies/normalized_entropy.jl

```diff
@@ -0,0 +1,80 @@
+export entropy_normalized
+
+"""
+    entropy_normalized(f::Function, x, est::ProbabilitiesEstimator, args...; kwargs...)
+
+Convenience syntax for normalizing to the entropy of uniform probability distribution.
+First estimates probabilities as `p::Probabilities = f(x, est, args...; kwargs...`),
+then calls `entropy_normalized(f, p, args...; kwargs...)`.
+
+Normalization is only defined for estimators for which [`alphabet_length`](@ref) is defined,
+meaning that the total number of states or symbols is known beforehand.
+
+    entropy_normalized(f::Function, p::Probabilities, est::ProbabilitiesEstimator, args...;
+        kwargs...)
+
+Normalize the entropy, as returned by the entropy function `f` called with the given
+arguments (i.e. `f(p, args...; kwargs...)`), to the entropy of a uniform distribution,
+inferring [`alphabet_length`](@ref) from `est`.
```

Comment:
I'm not convinced that this method is useful.... If the user has the `est` they may as well use the first method.

Reply:
We can just remove it. The user can just normalize to `log(N)` manually if they need to.

Reply:
Wait, I'm not so sure of that. If we remove this method, then:

Say I have a set of probabilities `p = Probabilities([0.2, 0.2, 0.3, 0.3])` *that I have already computed with a known estimator, but I don't have the original data for*. I now want to compute 1) Renyi entropy and 2) normalized Renyi entropy. Without this method, it is easy to compute 1), but no convenient way to compute 2), without re-implementing precisely this function does myself. That is: I'd always have to start from raw data if wanting to compute normalized entropy.

Reply:
What do you mean, you do `entropy_renyi(p)/log(length(p))`. What's the problem :D

Reply:
> What do you mean, you do entropy_renyi(p)/log(length(p)). What's the problem :D

[See this comment](https://github.com/JuliaDynamics/Entropies.jl/pull/99#discussion_r980261798).